libs/external-db-mysql/src/sql_schema_translator.spec.ts

```diff
@@ -124,10 +124,14 @@ describe('Sql Schema Column Translator', () => {
 
         describe('other fields', () => {
             test('boolean', () => {
-                ['tinyint'].forEach(t => {
+                ['tinyint', 'bit'].forEach(t => {
```

Comment:
`tinyint` to boolean - why?

Reply:
Not part of this PR but you're right, I'll fix it